.buildkite/docker-compose.yml

```diff
@@ -1,21 +1,7 @@
 version: '3.5'
 
 services:
-  e2e:
-    build:
-      context: .
-      dockerfile: Dockerfile-e2e
-    environment:
-      - BUILDKITE_BUILD_NUMBER
-      - BUILDKITE_JOB_ID
-      - BUILDKITE_PARALLEL_JOB
-      - BUILDKITE_PARALLEL_JOB_COUNT
-      - BUILDKITE_BUILD_ID
-      - BUILDKITE_SPLITTER_MODE            # splitting mode
-      - BUILDKITE_SPLITTER_BASE_URL        # url for fetching test plan
-      - BUILDKITE_SUITE_TOKEN
-      - ERROR_PLAN
-  build:
+  ci:
```

Comment:
change the service name from `build` to `ci` as we don't use this to build binaries any more.